### DISCUSSION TITLE
Share thoughts about DynControls

### DISCUSSION THREAD

**Discussion by matheo** [myndpm/open-source#27] [Ideas]
What do you think about the way `DynControls` extends the features and build their functionality?
https://mynd.dev/docs/dyn-forms/intro/dynamic-controls

**Comment by amitnil-dev** [comment]
I have messaged you on  discord/Dev.to post wherever possible to connect. Great work. Its really helpful and have been playing around the library.
I have a situation where I want to load entire form/form-controls  from JSON file(or say mongodb) metadata. Is there a way to do this or better approach perhaps. 

Again great work.
Thank you.
